RELEASE/dependencies.txt

```diff
@@ -1,7 +1,7 @@
-https://github.com/Ezandora/Detective-Solver/branches/Release/
-https://github.com/Ezandora/Helix-Fossil/branches/Release/
-https://github.com/Ezandora/Far-Future/branches/Release/
+github Ezandora/Detective-Solver Release
+github Ezandora/Helix-Fossil Release
+github Ezandora/Far-Future Release
+github gausie/excavator
 https://github.com/Ezandora/Voting-Booth/trunk/Release/
```

Comment:
Why not update voting and comb as well?

Reply:
they can't be without a manifest.json file since the scripts are in a subfolder